[spirv-reflect] missing include file in port.

**Describe the bug**
The spirv-reflect package builds OK, but does not include a new include file, necessary for using it:[include/spirv/unified1](https://github.com/KhronosGroup/SPIRV-Reflect/tree/master/include/spirv/unified1) 
Without this file, you cannot #include <spirv-reflect.h>, as intended.  Because it is missing; therefore the build will fail

**Environment**
- OS: Any
- Compiler: Any

**To Reproduce**
Steps to reproduce the behavior:
1. ./vcpkg install spirv-reflect
2. #include <spirv_reflect.h>
3. Build will fail.

**Failure logs**
1>C:\dev\vklive\vcpkg\installed\x64-windows-static-md\include\spirv_reflect.h(36,10): fatal  error C1083: Cannot open include file: './include/spirv/unified1/spirv.h': No such file or directory

**Additional context**
I believe this has been added to the spirv-reflect, but the port is not updated to include the new header.

unofficial-spirv-reflect

> VCPKG has install the `spirv.h` file into the `include/spirv/unified1` folder by installing the `spirv-headers` port, this port is the dependencie of `spirv-reflect`. For fixing your issue, please add `/DSPIRV_REFLECT_USE_SYSTEM_SPIRV_H` as a compile option in your repro project. Also, this macro will be added automatically when you use this port by below command in a cmake project:
> 
> ```
>     find_package(unofficial-spirv-reflect CONFIG REQUIRED)
>     target_link_libraries(main PRIVATE unofficial::spirv-reflect::spirv-reflect)
> ```

This is not working after #35479.   
Failure logs:

CMake Error at Library/vcpkg/scripts/buildsystems/vcpkg.cmake:859 (_find_package):
[cmake]   Could not find a package configuration file provided by
[cmake]   "unofficial-spirv-reflect" with any of the following names:
[cmake] 
[cmake]     unofficial-spirv-reflectConfig.cmake
[cmake]     unofficial-spirv-reflect-config.cmake
[cmake] 
[cmake]   Add the installation prefix of "unofficial-spirv-reflect" to
[cmake]   CMAKE_PREFIX_PATH or set "unofficial-spirv-reflect_DIR" to a directory
[cmake]   containing one of the above files.  If "unofficial-spirv-reflect" provides
[cmake]   a separate development package or SDK, be sure it has been installed.
[cmake] Call Stack (most recent call first):
[cmake]   Party/CMakeLists.txt:38 (FIND_PACKAGE)

Also I cant find any usege info while "vcpkg install spirv-reflect:x64-windows"

--- a/ports/spirv-reflect/export-targets.patch
+++ b/ports/spirv-reflect/export-targets.patch
@@ -1,0 +1,28 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 040c818..0e1476c 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -111,8 +111,20 @@ if(SPIRV_REFLECT_STATIC_LIB)
+     add_library(spirv-reflect-static STATIC ${CMAKE_CURRENT_SOURCE_DIR}/spirv_reflect.h
+                                      ${CMAKE_CURRENT_SOURCE_DIR}/spirv_reflect.c)
+ 
+-    target_include_directories(spirv-reflect-static
+-                               PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
+-    install(TARGETS spirv-reflect-static LIBRARY DESTINATION lib)
++    target_include_directories(spirv-reflect-static INTERFACE $<INSTALL_INTERFACE:include/spirv-reflect>)
++                               
++    set_target_properties(spirv-reflect-static PROPERTIES EXPORT_NAME spirv-reflect)
++    
++    install(TARGETS spirv-reflect-static
++            EXPORT unofficial-spirv-reflect-config
++            LIBRARY DESTINATION lib)
++            
++    install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/include/spirv/unified1/spirv.h DESTINATION include/spirv-reflect)
++    install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/spirv_reflect.h DESTINATION include/spirv-reflect)
++    
++    install(EXPORT unofficial-spirv-reflect-config
++            FILE unofficial-spirv-reflect-config.cmake
++            NAMESPACE unofficial::
++            DESTINATION share/unofficial-spirv-reflect)
+ endif()
+ 

--- a/ports/spirv-reflect/portfile.cmake
+++ b/ports/spirv-reflect/portfile.cmake
@@ -5,21 +5,28 @@ vcpkg_from_github(
     REPO KhronosGroup/SPIRV-Reflect
     REF "vulkan-sdk-${VERSION}"
     SHA512 b09a9c6ea4fb28bf860026e0dd7a95db30c191da29a3fb936d80f9b95d1e1ac665e3a9a8941687eb4a8d1a85447209c718d310241d41c0779aef8d33851bc7ba
-    HEAD_REF master
+    HEAD_REF main
+    PATCHES
+        export-targets.patch
 )
 
 vcpkg_cmake_configure(
     SOURCE_PATH "${SOURCE_PATH}"
-    OPTIONS -DSPIRV_REFLECT_STATIC_LIB=ON
+    OPTIONS
+        -DSPIRV_REFLECT_STATIC_LIB=ON
+        -DSPIRV_REFLECT_EXAMPLES=OFF
+        -DSPIRV_REFLECT_BUILD_TESTS=OFF
 )
 
 vcpkg_cmake_install()
+
+vcpkg_cmake_config_fixup(PACKAGE_NAME unofficial-spirv-reflect)
+
+vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/include/spirv-reflect/spirv_reflect.h" "./include/spirv/unified1/spirv.h" "spirv.h")
 
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/share")
 
 vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE")
-
-set(VCPKG_POLICY_EMPTY_INCLUDE_FOLDER enabled)
 
 vcpkg_copy_tools(TOOL_NAMES spirv-reflect-pp spirv-reflect AUTO_CLEAN)

--- a/ports/spirv-reflect/vcpkg.json
+++ b/ports/spirv-reflect/vcpkg.json
@@ -1,11 +1,11 @@
 {
   "name": "spirv-reflect",
   "version": "1.3.280.0",
+  "port-version": 1,
   "description": "SPIRV-Reflect is a lightweight library that provides a C/C++ reflection API for SPIR-V shader bytecode in Vulkan applications.",
   "homepage": "https://github.com/KhronosGroup/SPIRV-Reflect",
   "license": "Apache-2.0",
   "dependencies": [
-    "spirv-headers",
     {
       "name": "vcpkg-cmake",
       "host": true

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -8318,7 +8318,7 @@
     },
     "spirv-reflect": {
       "baseline": "1.3.280.0",
-      "port-version": 0
+      "port-version": 1
     },
     "spirv-tools": {
       "baseline": "1.3.280.0",

--- a/versions/s-/spirv-reflect.json
+++ b/versions/s-/spirv-reflect.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "f5cfadd5ead0c5a5e825e2e21b314bc902a43261",
+      "version": "1.3.280.0",
+      "port-version": 1
+    },
+    {
       "git-tree": "dbbfffe0c147230e88f43f57d77b34c59f29b3f6",
       "version": "1.3.280.0",
       "port-version": 0

